ui/apps/platform/src/Containers/Policies/Modal/ImportPolicyJSONModal.tsx

```diff
@@ -48,20 +48,27 @@ function ImportPolicyJSONModal({
             .then((response) => {
                 if (response.allSucceeded) {
                     setModalType('success');
+
                     // TODO: multiple policies import will be handled in
-                    // https://stack-rox.atlassian.net/browse/ROX-8613
+                    // https://issues.redhat.com/browse/ROX-8613
                     setPolicies([response.responses[0].policy]);
                     setTimeout(() => {
                         handleCancelModal();
                         fetchPoliciesWithQuery();
-                    }, 3000);
+                    }, 4000);
                 } else {
                     const errors = parsePolicyImportErrors(response?.responses);
                     const onlyHasDupeErrors = checkDupeOnlyErrors(errors);
-                    if (onlyHasDupeErrors) {
+                    if (policiesToImport.length === 1 && onlyHasDupeErrors) {
                         setDuplicateErrors(errors[0]);
                     }
-                    const errorMessageArray = getErrorMessages(errors[0]).map(({ msg }) => msg);
+
+                    // the errors array in the response is a single-element array,
+                    //     that contains an array with as many elements as there are policies in the import file
+                    //     and each of those elements is an array of error objeccts
+                    // hence, we use .flat() to un-ravel that structure to get all the errors
+                    const errorMessageArray = getErrorMessages(errors.flat()).map(({ msg }) => msg);
```

Comment:
i suppose this doesn't hurt -- we were only showing the first one anyways since we're forcing a single import